### PR TITLE
dev-python/shtab: new package, add 1.7.0

### PR DESCRIPTION
**Pull request by MocioF**
Dependency needed by >=greenbone-feed-sync-24.1.1

### CODE REVIEW

File: dev-python/shtab/shtab-1.6.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	https://pypi.org/project/shtab/
+"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="dev-python/setuptools-scm[${PYTHON_USEDEP}]"
+
+PATCHES=( "${FILESDIR}/${P}-pyproject.patch" )
```
Comment (mgorny) [inline dev-python/shtab/shtab-1.6.5.ebuild:23]: Did you submit the patch upstream?

File: dev-python/shtab/shtab-1.6.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} pypy3 )
```
Comment (mgorny) [inline dev-python/shtab/shtab-1.6.5.ebuild:7]: Any reason not to support 3.10? It's kinda weird to have PyPy3.10 but not CPython 3.10 here.

File: dev-python/shtab/shtab-1.6.5.ebuild (hunk 1 above)
Comment (pastalian) [reply dev-python/shtab/shtab-1.6.5.ebuild:23]: sent upstream. https://github.com/iterative/shtab/pull/162
@MocioF Can you add this link to the patch header?

Comment (mgorny) [reply dev-python/shtab/shtab-1.6.5.ebuild:23]: Add it as a comment inside the `PATCHES` array, please, so we can see it immediately when looking inside the ebuild file.

File: dev-python/shtab/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
```
Comment (pastalian) [inline dev-python/shtab/metadata.xml:4]:
```suggestion
	<maintainer type="person" proxied="yes">
```
I think it should tag me as a proxied maintainer.

**Comment by pastalian** [comment]
ACK, thanks for the bump.

**Pull request closed without merging**